apps/studio/src/schemas/common.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod"
+
+export const generateBasePermalinkSchema = (label: string) =>
+  z
+    .string({
+      required_error: `Enter a URL for this ${label}.`,
+    })
+    // Using `*` instead of `+` to allow empty strings, so the correct required error message is shown instead of the regex error message.
+    .regex(/^[a-z0-9\-]*$/, {
```

Comment:
Is there a minimum length? Cos this regex allows 0 chars.
```suggestion
    .regex(/^[a-z0-9\-]+$/, {
```

Reply:
yeap it's actually intended so we show the proper error message! is this a security issue?